packages/cli/src/lib/walker-package-ranger.js

```diff
@@ -257,9 +257,10 @@ async function walkPackageJson(packageJson = {}, walkedPackages = new Set()) {
         const resolvedRoot = derivePackageRoot(resolved);
 
         if (resolvedRoot) {
-          const resolvedPackageJson = (
-            await import(new URL("./package.json", resolvedRoot), { with: { type: "json" } })
-          ).default;
+          const resolvedPackageJson =
+            // @ts-expect-error see https://github.com/microsoft/TypeScript/issues/42866
+            (await import(new URL("./package.json", resolvedRoot), { with: { type: "json" } }))
```

Comment:
a .toString() here avoids the TS error about URLs instead of string

Reply:
This comment repeats in a few other places.  It's a very slight hack to do it that way, but avoiding the ts-expect-error prevents the disable from masking other potential errors on the same line.

Reply:
good to know, but I would prefer to not have to alter code just to fit TS, especially when TS is wrong (in this case 😄 )